src/dex/uniswap-v3/scripts/check-pool-bitmap-ranges.ts

```diff
@@ -0,0 +1,238 @@
+// /*
+//  * It is just standalone helper script to estimate how wide bitMap range can be.
+//  * We need it in order to cover all possible state variations
+//  */
+// import * as dotenv from 'dotenv';
+// dotenv.config();
```

Comment:
Why it is commented? It is not imported anywhere and will not harm. Maybe we even don't need it anymore. But I think better to keep it

Reply:
Compile error I needed to handle that